packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -23,7 +23,7 @@ import '../test/runner.dart';
 import '../test/test_wrapper.dart';
 import '../test/watcher.dart';
 
-class TestCommand extends FlutterCommand {
+class TestCommand extends FlutterCommand with DeviceBasedDevelopmentArtifacts {
```

Comment:
Consider extending the description of the test command to describe the difference in behavior between test and integration test commands

Reply:
Updated the doc comment for this